webhooks_skeleton.js

```diff
@@ -0,0 +1,10 @@
+const sqlite3 = require('sqlite3')
```

Comment:
Since there's no `package.json` file we'll have to remember to tell candidates to `npm install sqlite3` before they attempt to run this. Maybe we should turn this into a full node project to makes things a little more intuitive? That way they just clone, do the usual `npm install` and they're good to go.

Thoughts on that? Since there's only one dependency it doesn't gain us much... I could go either way!